test/helpers/bitcoin-core.ts

```diff
@@ -33,6 +33,26 @@ export async function sendToAddressAndConfirm({
   await walletClient.generateToAddress({ nblocks: 6, address: RANDOM_ADDRESS })
 }
 
+export async function sendToAddress({
```

Comment:
helpers began previous to define some code rules but this should be
```
export const sendToAddress = ...
```

Reply:
Done: https://github.com/GaloyMoney/galoy/pull/1511/commits/b718f72b7ca6eebb857141bd31aa67dd11601b7a